_pages/en/strategy-build-vs-buy.md

```diff
@@ -110,26 +110,27 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 ### Governance, Compliance, and Reporting
 
 1. IT Solutions are broken down into manageable parts (IT Products)
-2. Each IT Product must be assessed on its own as well as a part of overall the IT Solution.
-3. IT Products may be bought, built, or adopted (e.g. Open Source Software)
-4. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).
+2. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).
   As such:
    1. Complex and core to mandate IT Solutions are architected and delivered internally by ESDC employees but may leverage third party IT Products as components.
    2. Non-complex and not core to mandate IT Solutions may be architected and delivered by third party integrators and/or vendors.
-5. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
+3. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
    - Accessibility test results
    - Integration testing results
    - Security test results
    - API consumption statistics
    - Production deployment audit traces
    - Availability statistics
-6. Security Assessment & Authority (SA&A) uses the Target-SA&A methodology
-7. Accessibility Assessment uses the Target-Accessibility methodology
-8. Audit Assessment uses the Target-Audit methodology
-9. IT Solutions reuse existing IT Products, including:
+4. Security Assessment & Authority (SA&A) uses the Target-SA&A methodology
+5. Accessibility Assessment uses the Target-Accessibility methodology
+6. Audit Assessment uses the Target-Audit methodology
```

Comment:
Targit-Audit methodology, Target-Accessibility methodology 

Are these well known such that we don't need to reference them ? Or is referencing them not relevant  in this context

Reply:
Hmm ... this sentence is verbatum in the [Target Operating Model](http://localhost:4000/ITStrategy/strategy-target-solution-delivery-model.html) document is well?

It's even the same number ... 

6. Audit Assessment uses the Target-Audit methodology

5 and 4 are the same too ... is this a WIP thing, or is that intentional?